assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -367,11 +334,7 @@
 		--sidebar-menu-hover-background-gradient: #{hex-to-rgb( $muriel-white )};
 		--sidebar-menu-hover-color: #{$muriel-blue-500};
 
-		// TODO: once we replace the current default theme with Classic Bright, we can remove these and increase consistency by using the sidebar colors instead
-		--button-is-borderless-color: #{$gray-dark};
-		--count-border-color: #{$gray-dark};
-		--count-color: #{$gray-dark};
-		--profile-gravatar-user-secondary-info-color: #{$gray-dark};
+		// TODO: once we replace the current default theme with Classic Bright, we can remove these and increase consistency by using the sidebar colors instead								
```

Comment:
The default theme is Classic Bright now, so unsure about what to do here now. @flootr

Reply:
I think that's an issue on its own. Check whether all those variables are still needed.

Reply:
Each one is only used once, but I think they still are if the intention is to keep the colours the same. I don't really understand the note though. There's no other variable which can replace both of them whilst keeping both schemes the same, I'm pretty sure...unless I'm misinterpreting or missing something.